src/prommis/nanofiltration/costing/diafiltration.py

```diff
@@ -0,0 +1,713 @@
+#########################################################
```

Comment:
1. There is a PrOMMiS header that should be included in every file (just copy it from another file).

2. For documentation (and pylint) purposes, it is good to have a short doc string at the start of each file describing what it does, etc. For this case, the existing comment could be part of it, but it should be put inside a block quote (`"""`).

3. You should add these modules to `/docs/api.rst` so that they appear in the autodocs.